app/javascript/components/volunteer/registration/Step2.jsx

```diff
@@ -128,11 +130,11 @@ class Step2 extends React.Component {
         <section>
           <label htmlFor="skills">
             <h3>Skills</h3>
-            <Field
-              name="skills"
-              component="textarea"
-              row="6"
-              style={{ resize: "none" }}
+            <Dropdown
```

Comment:
We could get away without passing down this `toggleSelected` function if we use a custom input for a Formik field. Check out what I did with `PhoneInput` [here](https://github.com/calblueprint/rising-tides/pull/157/files#diff-8f33d6bbbbe2e70c1acc75f84b2400a3R29-R46) and see if you can do something similar!